src/components/event/experimental-api/ExperimentalAPIEventStore.ts

```diff
@@ -109,12 +113,18 @@ export class ExperimentalAPIEventStore {
 		if (this.isLoadingChildren.get(eventId) || this.hasMoreChildren.get(eventId) === false) return;
 		const currentChildren = this.parentsChildrenMapMap.get(eventId) || [];
 		try {
-			const ids = await api.events.getChildrenIds({ parentId: eventId });
+			const ids = await api.events.getChildrenIds({
+				parentId: eventId,
+				limit: this.CHILDREN_COUNT_LIMIT + 1,
+				offset: (this.parentsChildrenChunks.get(eventId) || 0) * this.CHILDREN_COUNT_LIMIT,
```

Comment:
I think we can just use `parentsChildrenMapMap.get(id).length || 0` and remove this extra state